examples/machine/im_direct_mpc_curr_ctr.py

```diff
@@ -1,35 +1,39 @@
-#pylint: disable=wrong-import-position
+"""
+Example of controlling an induction machine with direct model predictive control MPC.
```

Comment:
Example of direct model predictive control (MPC)  for an induction machine drive system. The controller aims to track the stator current reference calculated based on the reference values of the stator flux magnitude and torque. The machine operates at a constant (nominal) speed.

Reply:
Resolved in the next commit